src/pytorch_lightning/core/module.py

```diff
@@ -1193,10 +1193,6 @@ def configure_callbacks(self):
                 early_stop = EarlyStopping(monitor="val_acc", mode="max")
                 checkpoint = ModelCheckpoint(monitor="val_loss")
                 return [early_stop, checkpoint]
-
```

Comment:
Not sure whether this should be removed, or a different example should be given.